### ISSUE TITLE
Special characters in filenames of images uploaded through parse dashabord

### ISSUE THREAD

**Issue by JonComo** [ParsePlatform/parse-dashboard#573]
#### Environment Setup

Standalone parse-server with s3 filesAdapter, iOS parse SDK.

#### Steps to reproduce

Not sure if this has been seen by others before, but thought it was interesting to note. I was getting  403 forbidden when trying to load an image posted through the dashboard. It turns out that having a space character in the filename was the issue. The file is still correctly saved in s3, and is publicly accessible, its just the iOS request that's failing. 

When all spaces are removed before uploading through the dashboard, the file request works correctly without 403 error.

#### Logs/Trace

from iOS, when filename contains a space:
Response status code was unacceptable: 403 (Code: 1, Version: 1.13.0)

Haven't looked into a solution yet, just wanted to bring this to your attention as I had to do a bit of trial and error to see where the problem originated. Thanks!

**Comment by mario** [comment]
Yup, this is a real problem - probably a bug in Parse server itself tho. Let me know if you get the time to investigate.
